helix-core/src/textobject.rs

```diff
@@ -181,10 +185,10 @@ mod test {
                     (14, Inside, (14, 21)),
                     (20, Inside, (14, 21)),
                     (17, Inside, (14, 21)),
-                    (14, Around, (13, 22)),
+                    (14, Around, (14, 21)),
                     // FIXME: edge case
                     // (20, Around, (14, 20)),
```

Comment:
Should this be updated?

Reply:
You mean the `FIXME`-commented case?  It probably should be updated, but I wasn't sure what the intended behavior was.  I assume it should result in `(14, 21)` like everything else there, but it doesn't seem like an edge case to me, so the comment left me a little confused.

Reply:
Went ahead and updated and uncommented that test case.  Thanks for the poke!